all-core/gl/conversions.go

```diff
@@ -46,6 +48,13 @@ func PtrOffset(offset int) unsafe.Pointer {
 	return unsafe.Pointer(uintptr(offset))
 }
 
+// StrFree takes a Go string and allocates a C string in the C heap using malloc.
+// It is the caller's responsibility to free it by calling returned free.
+func StrFree(str string) (cstr *uint8, free func()) {
```

Comment:
For better documentation. I thought if it returned `(*uint8, func())` would be less clear what 2nd value is.